sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/FaultInjectionWithAvailabilityStrategyTests.java

```diff
@@ -747,6 +763,49 @@ public Object[][] testConfigs_readAfterCreation() {
                 validateStatusCodeIsInternalServerError,
                 validateDiagnosticsContextHasDiagnosticsForOnlyFirstRegion
             },
+            // This test injects 429 (Request_Rate_Too_Large) into the local region only.
+            // No availability strategy exists - expected outcome is request failed due to 429
+            // SDK does not do cross region retry for 429 by default
+            new Object[] {
```

Comment:
This is outside the scope of this PR, but I wonder if we can add cross-regional retries for 429.

Reply:
This is a good question ,429 will not trigger cross region retry by default, maybe for certain situation it can help, worth discussion more about it

Reply:
I would argue that we should not auto-retry on 429 across region. Has the risk of "leaking" high load on hot partitions into other regions as well if something wild is happening. I would also consider this a breaking behavior change. But worth discussing in Java-sync next Monday.